Branch-SDK/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     android:versionCode="1"
     android:versionName="1.0" >
+    <uses-permission android:name="android.permission.READ_LOGS"
```

Comment:
This permission is not appropriate for us to declare as it goes beyond our scope https://developer.android.com/reference/android/Manifest.permission#READ_LOGS

Reply:
Removed, thanks @gdeluna-branch !